src/core.js

```diff
@@ -303,6 +305,7 @@ function done() {
 	passed = config.stats.all - config.stats.bad;
 
 	runLoggingCallbacks( "done", {
+		skipped: Test.count - config.stats.all,
```

Comment:
This deserves some tests. And the tests might be the most complex to implement this single line. Maybe we should split this part to address it in another PR.